en/asgardeo/docs/references/service-extensions/in-flow-extensions/custom-authentication/api/custom-authentication-action-v1.yaml

```diff
@@ -428,8 +432,8 @@ components:
           format: uri
           description: Claim uri defined in the 'http://wso2.org/claims' dialect
         value:
-          type: string
-          description: Claim value
+          type: object
+          description: Claim value. If the claim is multi-valued, it should be a list of strings (the `,` is not allowed in claim values). Otherwise, it should be a single string.
```

Comment:
Shall we say 'Note the ',' is not allowed in claim values as it is used internally to process data.